pkg/events/sorting/sorting_test.go

```diff
@@ -25,7 +25,7 @@ func (e sortableEventsList) Less(i, j int) bool {
 }
 
 func (e sortableEventsList) Swap(i, j int) {
-	return
+
```

Comment:
I know it's only used in sort.IsSorted, but if one day we use this sort.Interface implementation to do real sorting we can waste a bit of time troubleshooting. So I'd completed the implementation as follows:
```suggestion
	e.eventsList[i], e.eventsList[j] = e.eventsList[j], e.eventsList[i]
```
Or panic("not implemented")